containers/release-candidate/src/daffodil-release-candidate

```diff
@@ -300,6 +301,23 @@ case $PROJECT_REPO in
     yarn package
     cp *.vsix $DAFFODIL_RELEASE_DIR/bin/
 
+    echo "Building Extension Docs..."
+    git clone https://github.com/apache/daffodil-vscode.wiki.git .wiki
+    for markdown_file in $(echo "\
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Development \
+      Apache-Daffodil™-Extension-for-Visual-Studio-Code:-Roadmap \
+      Home"
+    ); do
+      pandoc -t html -f markdown -o "${markdown_file}.html" ".wiki/${markdown_file}.md"
+    done
+    rm -rf .wiki
+
+    echo "Installing Extension Docs..."
+    rm -rf $DAFFODIL_VSCODE_DOCS_DIR
+    mkdir -p $DAFFODIL_VSCODE_DOCS_DIR
+    cp -R *.html $DAFFODIL_VSCODE_DOCS_DIR
+
```

Comment:
Below under "Committing Site Changes", we have a line that looks like this:
```
git diff-index --quiet HEAD || git commit -m "Stage release v$VERSION-$PRE_RELEASE"
```
Since this now could commit something for daffodil or daffodil vscode, I suggest we change it to this:
```
git diff-index --quiet HEAD || git commit -m "Stage $PROJECT_NAME release v$VERSION-$PRE_RELEASE"
```